lib/node_modules/@stdlib/math/base/special/besselj0/test/test.js

```diff
@@ -138,7 +138,7 @@ tape( 'the function evaluates the Bessel function of the first kind of zero orde
 			t.equal( y, expected[i], 'x: '+x[i]+', y: '+y+', expected: '+expected[i] );
 		} else {
 			delta = abs( y - expected[i] );
-			tol = 130.0 * EPS * abs( expected[i] );
+			tol = 131.4 * EPS * abs( expected[i] );
```

Comment:
I had to tweak the tolerance levels here and below, by a small bit, i.e., from `130.0` to `131.4`, to match with the tests of the `C` implementation.